site-src/guides/metrics.md

```diff
@@ -116,6 +116,54 @@ kubectl -n default port-forward inference-gateway-ext-proc-pod-name  9090
 curl -H "Authorization: Bearer $TOKEN" localhost:9090/metrics
 ```
 
+## Setting Up Grafana + Prometheus
+
+### Grafana
+
+A simple grafana deployment can be done with the following commands:
+
+```bash
+     helm repo add grafana https://grafana.github.io/helm-charts
+     helm install grafana grafana/grafana --namespace monitoring
```

Comment:
remove extra spaces:
```suggestion
helm repo add grafana https://grafana.github.io/helm-charts
helm install grafana grafana/grafana --namespace monitoring
```

same for other places.

Reply:
btw, this is how it looks like with the spaces:
<img width="856" alt="image" src="https://github.com/user-attachments/assets/ab74646c-1f67-4b2b-878b-df9327274923" />